src/app/leverage/components/your-tokens.tsx

```diff
@@ -17,9 +25,10 @@ const leverageTypeLabels = {
 }
 
 export function YourTokens() {
-  const { chainId } = useNetwork()
+  const { chain } = useAccount()
```

Comment:
Could still use `chainId` from `useNetwork` here? We're actually actively trying to avoid scattering too many library imports throughout the app in case we ever wanna change out a lib again e.g. ethers -> viem. So that's why we have these wrapper functions like `useNetwork`